src/main/java/com/assignment6/hello/Hiroshi.java

```diff
@@ -0,0 +1,28 @@
+package com.assignment6.hello;
+
+
+
+public class Hiroshi {

```

Comment:
たくさんのヒロシを管理するならこれでもよいですが、
```suggestion
public class CrayonShinchanCharacter {

```
とかでもいいかも。

Reply:
ご指摘いただいた通り、たくさんのひろしを管理するのは大変だし、ほかのキャラクターが出てきたときのことも考えてご提案いただいたものにしてみました！
＜今回の修正部分のコミット＞
ab92db267ada6d37212bd0c8d14f9226698a8e49

Reply:
okです！